Add documentation for new nox commands (#981)
ethyca/fidesops: docs/fidesops/docs/ui/local_dev.md

## Backend deployment

> Fidesops automatically serves a version of the UI when running `docker-compose up`.

Reviewer: ```suggestion
Fidesops automatically serves a version of the UI when running `nox -s dev`.
```

This will build and place the Admin UI files into a location accessible by backend fidesops deployments.

> To test the UI, run `docker-compose up` from the root directory, and visit `http://0.0.0.0:8080/static/index.html`.

Reviewer: ```suggestion
To test the UI, run `docker-compose up webserver` from the root directory, and visit `http://0.0.0.0:8080/static/index.html`.
```

> To test the UI, run `docker-compose up` from the root directory, and visit `http://0.0.0.0:8080/static/index.html`.

Reviewer 2: That is correct. The UI is now served from `http://0.0.0.0:8080/`.

> To test the UI, run `docker-compose up webserver` from the root directory, and visit `http://0.0.0.0:8080/index.html`.

Reviewer: ```suggestion
To test the UI, run `nox -s dev` from the root directory, and visit `http://0.0.0.0:8080/index.html`.
```
